libcloud/storage/drivers/backblaze_b2.py

```diff
@@ -301,8 +325,13 @@ def upload_object(self, file_path, container, object_name, extra=None,
         # Note: We don't use any of the base driver functions since Backblaze
         # API requires you to provide SHA1 has upfront and the base methods
         # don't support that
-        with open(file_path, 'rb') as fp:
-            iterator = iter(fp)
+        object_name = object_name.replace('\\', '/')
```

Comment:
Might be a good idea to put this into a utility method - `escape/sanitize_object_name` or similar.